src/pages/Environments.js

```diff
@@ -135,24 +136,33 @@ const Environments = () => {
 
   const refreshData = Utils.withBusyState(setLoading, async () => {
     const creator = getUser().email
+
+    const startTimeForLeoCallsEpochMs = new Date().getTime()
```

Comment:
FYI, [`Date.now`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/now) is a more concise way to get a timestamp.